examples/lock-app/nrfconnect/README.md

```diff
@@ -277,17 +277,9 @@ To debug the application on target:
         $ cd <example-dir>
         $ west debug
 
-<a name="accessing-the-command-line"></a>
+## Testing the example
 
-## Accessing the command line
+Check the [CLI tutorial](../../../docs/guides/nrfconnect_examples_cli.md) to
+learn how to use command-line interface of the application.
 
-The application includes a command line interface with support for logs and the
-OpenThread commands.
-
-To access it, use any serial terminal program you like, for example `minicom` or
-`GNU screen`.
-
-The UART interface is configured for `115200` baud rate.
-
-All OpenThread commands must be prefixed with `ot`, for example
-`ot thread start`.
+TODO: mention Rendezvous tutorial here
```

Comment:
Don't mention TODO in docs. Preferred approach: create a JIRA to track this.

Reply:
OK, will remove